src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -10,66 +15,70 @@ export module DotNet {
   const pendingDotNetToJSStreams = new Map<number, PendingStream>();
 
   const jsObjectIdKey = "__jsObjectId";
-  const dotNetObjectRefKey = '__dotNetObject';
-  const byteArrayRefKey = '__byte[]';
-  const dotNetStreamRefKey = '__dotNetStream';
+  const dotNetObjectRefKey = "__dotNetObject";
+  const byteArrayRefKey = "__byte[]";
+  const dotNetStreamRefKey = "__dotNetStream";
   const jsStreamReferenceLengthKey = "__jsStreamReferenceLength";
 
   class JSObject {
-    _cachedFunctions: Map<string, Function>;
+      _cachedFunctions: Map<string, Function>;
 
-    constructor(private _jsObject: any) {
-      this._cachedFunctions = new Map<string, Function>();
-    }
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      constructor(private _jsObject: any) {
+          this._cachedFunctions = new Map<string, Function>();
+      }
 
-    public findFunction(identifier: string) {
-      const cachedFunction = this._cachedFunctions.get(identifier);
+      public findFunction(identifier: string) {
+          const cachedFunction = this._cachedFunctions.get(identifier);
 
-      if (cachedFunction) {
-        return cachedFunction;
-      }
+          if (cachedFunction) {
+              return cachedFunction;
+          }
 
-      let result: any = this._jsObject;
-      let lastSegmentValue: any;
+          let result: any = this._jsObject;
+          let lastSegmentValue: any;
 
-      identifier.split('.').forEach(segment => {
```

Comment:
Note; reviewing this file with `?w=1` may be easier given indentation / formatting changes.